torch/_inductor/lowering.py

```diff
@@ -3211,6 +3215,40 @@ def load_source_val():
     return view(result_flat, self.get_size())
 
 
+@register_lowering(aten._unsafe_masked_index, type_promotion_kind=None)
+def _unsafe_masked_index(self, mask, indices, fill):
+    ranges, _unsafe_index_fn = index_impl_helper(self, indices, check=False)
+    mask_loader = mask.make_loader()
+
+    def inner_fn(idx):
+        return ops.masked(mask_loader(idx), lambda: _unsafe_index_fn(idx), fill)
+
+    return Pointwise.create(
+        device=self.get_device(),
+        dtype=self.get_dtype(),
+        inner_fn=inner_fn,
+        ranges=ranges,
+    )
+
+
+@register_lowering(aten._unsafe_masked_index_put_accumulate)
+def _unsafe_masked_index_put_accumulate(x, mask, indices, values):
+    masked_value = where(mask, values, 3)
```

Comment:
Is this to catch errors? If so, can you leave a comment?

Reply:
Sorry about that. I'll be more careful to not push debug stuff.

Reply:
It's alright. I didn't dislike the idea of using a funny value here to catch errors. In any case, either is fine, as eventually this will be replaced by a masked atomic_add.